### ISSUE TITLE
RTL Languages Support in feeds

### ISSUE THREAD

**Issue by 183amir** [cdhigh/KindleEar#205]
Hi, First of all thank you for this great app! :)

I am trying to read some feeds in Persian but looks like the generated content does not support RTL languages.
When I use the Send to Kindle extension, it seems to support RTL languages. For example try converting [this page](http://www.bbc.co.uk/persian/world/2015/06/150607_l10_turkey_election_results).

**Comment by 183amir** [comment]
Hey I just noticed that this happens because the ebook is in `mobi` format while `azw3` format has no problem with RTL languages. Can I convert the output from `mobi` to `azw3` format before sending it to my kindle?

**Comment by cdhigh** [comment]
#6
mail service of Amazon don't accept azw3.

**Comment by 183amir** [comment]
How about `pdf`? I converted the output of KindleEar with Calibre to `pdf` and sent it with email without asking it to be converted. It seems to work very well this way. Maybe we can push both the `mobi` and `pdf` file? This way, I can read my English news on the `mobi`format and Persian news on `pdf` format.

**Comment by cdhigh** [comment]
pdf is much complicate, I don't have much time to consider and improve it.
Or other guys can help me to do it?

**Comment by 183amir** [comment]
I can do it myself but I need to know where to start at. There are so many python codes in your project and I cannot find the relevant codes.

**Comment by cdhigh** [comment]
apps/Work/Worker.py

**Comment by 183amir** [comment]
Hey, I have just noticed that fabreadly fixed this problem. They have changed their format but I am not sure which format they use yet.

**Comment by 183amir** [comment]
It's `azw3`!

**Comment by 183amir** [comment]
Ok. based on [this link](http://www.mobileread.com/forums/showthread.php?t=229170), I tried sending the both format; it was delivered but it showed me the old format and the file was renamed to `.azw` (while I sent an `.mobi` file). I also tried renaming the `.azw3` file to `.mobi` and send it by email but its delivery was rejected.

However, the file delivered by fabreadly has the `.azw3` extension and RTL works there.
